docs/docs/code-of-conduct.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Gatsby Contributor Covenant Code of Conduct
+title: Gatsby Contributor Covenant 
```

Comment:
Why was "Code of Contact" removed? That should be there.

Reply:
oh whoops! I had accidentally done that and thought I reversed it. Just fixed it!